packages/search/addon/components/navi-search-result/report-and-dashboard.hbs

```diff
@@ -0,0 +1,7 @@
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<h1>{{this.title}}</h1>
+<ul class="search-result">
+    {{#each @data as |result|}}
+    <li>{{result.title}}</li>
```

Comment:
Can we give a class to the `li` elements?

Reply:
also indent is off? on line 4